### PR TITLE
I have updated my profile - HareerSakhi to AvidCoder101

### PR DESCRIPTION
**Pull request by AvidCoder101**
I have just updated my section because I have changed some things on my README.md. Thanks.

### CODE REVIEW

**Review by bishalr0y — APPROVED** [review]

**Comment by AvidCoder101** [comment]
I have changed my name to the bottom of the file. @schmelto or @kaiwalyakoparkar I think that this PR might be ready to merge? Thank you.

File: data.json [hunk 1]
```diff
@@ -1162,5 +1156,11 @@
     "githubUsername": "siddhi-244",
     "imageUrl": "https://user-images.githubusercontent.com/69195262/126054873-11415026-bcee-40d4-81dc-db3076965670.jpg",
     "issueNumber": 598
+  },
+   {
```
Comment (schmelto) [inline data.json:1160]:
```suggestion
  {
```

**Review by schmelto — CHANGES_REQUESTED** [review]
thanks for adjusting the pr.
Only one small thing

File: data.json (hunk 1 above)
Comment (AvidCoder101) [reply data.json:1160]: Ok, fixing this now

Comment (AvidCoder101) [reply data.json]: Fixed this, thanks for the feedback

**Review by schmelto — APPROVED** [review]
lgtm

**Pull request merged**